frb_rust/src/handler.rs

```diff
@@ -321,7 +321,10 @@ impl ErrorHandler for ReportDartErrorHandler {
     }
 
     fn handle_error_sync(&self, error: Error) -> WireSyncReturn {
-        wire_sync_from_data(error.message(), false)
+        match error {
+            Error::CustomError(err) => wire_sync_from_data(err.box_into_dart(), false),
+            Error::Panic(err) => wire_sync_from_data(error_to_string(&err), false),
+        }
```

Comment:
```suggestion
let data =         match error {
            Error::CustomError(err) =>err.box_into_dart()
            Error::Panic(err) =>error_to_string(&err)
        };
        wire_sync_from_data(data, false)
```

Reply:
it won't work, there are different types
CustomError - wire_sync_from_data<DartAbi>
Panic - wire_sync_from_data<String>

Reply:
We can `wire_sync_from_data(error.into_dart(), false)`